webapp/src/actions/index.js

```diff
@@ -80,4 +80,34 @@ export function performSearch(terms) {
 
         return dispatch(searchPostsWithParams(teamId, {terms, is_or_search: false, include_deleted_channels: viewArchivedChannels, page: 0, per_page: 20}, true));
     };
-}
\ No newline at end of file
+}
+
+export function requeueItem(itemId) {
+    return async (dispatch, getState) => {
+        const command = `/agenda requeue ${itemId}`;
```

Comment:
What is `itemId`? It seems `requeueItem` is called like this:

```js
registry.registerPostDropdownMenuAction('Re-queue', (params) => {
    store.dispatch(requeueItem(params));
});
```

which would mean `params` is an `itemId`?

Reply:
Will the user ever run `/agenda requeue` manually? If not, we should use an http endpoint for this feature and create a slash command. If the user would use this command manually, what would they supply for the `itemId`?

Reply:
Sure will refactor `params` to `itemId`.
Made `/agenda requeue` to support future options like
`/agenda requeue all` # meeting canceled, I want to requeue all items to next meeting.
`/agenda requeue last` # requeue last items
`/agenda requeue last n` # requeue last n items

Reply:
Okay if this is the case, then maybe we should have a `/agenda requeue item (itemId)` command, and use that for the current programmatic case. Also what is `itemId`? Is it the post ID? Let's use `postId` as the identifier instead of `itemId` if this is the case.

Reply:
Right, will change it to `postId` and change command as well.